packages/@react-spectrum/progress/src/ProgressBarBase.tsx

```diff
@@ -86,7 +86,7 @@ function ProgressBarBase(props: ProgressBarBaseProps, ref: DOMRef<HTMLDivElement
       }
       {showValueLabel &&
         <div className={classNames(styles, 'spectrum-BarLoader-percentage')}>
-          {barProps['aria-valuetext']}
+          {barProps?.['aria-valuetext']}
```

Comment:
I'd add a check to line 89 for barProps existing, that way we don't render an empty percentage area